src/RestApi.ts

```diff
@@ -1,12 +1,15 @@
 import {compassLogger} from "./Logging";
 import * as $ from "jquery";
 
-const CONTENT_TYPE = 'application/vnd.iperity.compass.v2+json';
+// Compass content types, in decreasing order of preference
+const CONTENT_TYPES = [
+    'application/vnd.iperity.compass.v2+json',
+];
```

Comment:
I would say that, then, we should also add the new version to the array.
Right now it's a bunch of code that does nothing.

Reply:
At this point, there is no v3 in any Compass version, it's still pending as a MR...
This won't change probably until r-2020a.

Reply:
So when would you add it? When that version is deployed to at least one environment?

Reply:
Yes.